### ISSUE TITLE
It's possible embedded glowroot dist folder when build project ? 

### ISSUE THREAD

**Issue by J-Dudek** [glowroot/glowroot#953]
Hi,

I want to embed the Glowroot stack when I create my project.

Indeed, I do not want to manually put the dist folder on my server. I would like to integrate the stack when I deliver my service.

Thank you for your expertise.

**Comment by Never-lab** [comment]
Yes — treat the agent like any other runtime artifact in your delivery pipeline. There is no special “embed dist into Maven/Gradle so it auto-attaches” mode: you still need `-javaagent:/path/to/glowroot.jar` at JVM start.

Common patterns: copy the Glowroot dist (or just `glowroot.jar` + writable dir) into the Docker/image/deploy package and set the javaagent arg in the entrypoint / `JAVA_OPTS`. Same idea for systemd, Tomcat `setenv`, etc.

Wiki: [Agent Installation (Embedded)](https://github.com/glowroot/glowroot/wiki/Agent-Installation-(with-Embedded-Collector)). More Qs → [Discussions](https://github.com/glowroot/glowroot/discussions).

OK to close as support.

@nowheresly
